web-marketplace/src/components/organisms/PostForm/PostForm.types.ts

```diff
@@ -0,0 +1,4 @@
+export type PostFormPrivacyType =
+  | 'private'
+  | 'private-files'
+  | 'private-location';
```

Comment:
could you use the same string as in the db enum type? see https://github.com/regen-network/regen-server/blob/31d2eee881dbcede7bba4f3452e578b6ac85a82d/migrations/current.sql#L4